### PR TITLE
adding pmseqnum to eas payload TM-4033

### PR DESCRIPTION
**Pull request by SidelineCory24**
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2486)
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/354)

### CODE REVIEW

File: talentmap_api/fsbid/services/agenda_employees.py [hunk 1]
```diff
@@ -229,15 +229,16 @@ def fsbid_agenda_employee_to_talentmap_agenda_employee(data, cdos=[]):
...
         "hsAssignment": {
-            "orgDescription": pydash.get(data, "handshake[0].posorgshortdesc", None),
+            "orgDescription": pydash.get(data, "handshake[0].posorgshortdesc") or None,
         },
         "agenda": {
-            "panelDate": pydash.get(data, "latestAgendaItem[0].panels[0].pmddttm", None),
-            "status": pydash.get(data, "latestAgendaItem[0].aisdesctext", None),
+            "panelDate": pydash.get(data, "latestAgendaItem[0].panels[0].pmddttm") or None,
+            "status": pydash.get(data, "latestAgendaItem[0].aisdesctext") or None,
+            "pm_seq_num": pydash.get(data, "latestAgendaItem[0].panels[0].pmseqnum") or None,
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda_employees.py:241]: other values are camel case, do we want to keep consistent?

File: talentmap_api/fsbid/services/agenda_employees.py [hunk 2]
```diff
@@ -229,15 +229,16 @@ def fsbid_agenda_employee_to_talentmap_agenda_employee(data, cdos=[]):
             "cdo": cdo,
         },
         "currentAssignment": {
-            "TED": pydash.get(data, "currentAssignment[0].asgdetdteddate", None),
-            "orgDescription": pydash.get(data, "currentAssignment[0].position[0].posorgshortdesc", None),
+            "TED": pydash.get(data, "currentAssignment[0].asgdetdteddate") or None,
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda_employees.py:232]: `or None` 🥹

Comment (M00staff) [reply talentmap_api/fsbid/services/agenda_employees.py:232]: curious what the difference is here?

passing in `None` as the default (which i think is already the default return vs adding the `or None` at the end?

tried looking it up - but the wording on that google search doesn't return what i want it to

File: talentmap_api/fsbid/services/agenda_employees.py (hunk 1 above)
Comment (M00staff) [reply talentmap_api/fsbid/services/agenda_employees.py:241]: oh wow - y'all really mix it up here... my understanding was always snake_case for python and camelCase for javascript 

no point in going back and changing what is already merged and in production, but maybe try to stick to that going forward, for consistency?

Comment (SidelineCory24) [reply talentmap_api/fsbid/services/agenda_employees.py:241]: will update to 🐪

File: talentmap_api/fsbid/services/agenda_employees.py (hunk 2 above)
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/agenda_employees.py:232]: "(which i think is already the default return)" Yes! You'll see we've redundantly done that throughout our code, but are slowly changing to `or None` when we mess with related code. Passing in the default `None` (or nothing, as u noticed) will pass a `None` if the key is not found. What we are doing here is trying to save our FE a bit of type checking, so `or None` gives us a `None` if anything on the left side is falsey.

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/agenda_employees.py:232]: *not really type checking lol - just checking in general "if it's 0 or "" or None, then fallback"

**Review by elizabeth-jimenez — APPROVED** [review]
🐫

File: talentmap_api/fsbid/services/agenda_employees.py (hunk 1 above)
Comment (SidelineCory24) [reply talentmap_api/fsbid/services/agenda_employees.py]: updated FE as well

**Pull request merged**